### ISSUE TITLE
Building WebChat Errors

### ISSUE THREAD

**Issue by bpug** [Microsoft/BotFramework-WebChat#425]
Hi,

cannot build webchart

Could you help me please

node.js - 7.9.0
npm - 4.4.4

I become an error:

```
PS C:\Source\Repos\bot\BotFramework-WebChat> npm run build

> botframework-webchat@0.10.6 build C:\Source\Repos\bot\BotFramework-WebChat-ms
> tsc && npm run build-css && webpack --progress

src/Store.ts(82,9): error TS7018: Object literal's property 'carouselMargin' implicitly has an 'any' type.
src/Store.ts(121,9): error TS7018: Object literal's property 'width' implicitly has an 'any' type.
src/Store.ts(122,9): error TS7018: Object literal's property 'height' implicitly has an 'any' type.
src/Store.ts(161,9): error TS7018: Object literal's property 'botConnection' implicitly has an 'any' type.
src/Store.ts(162,9): error TS7018: Object literal's property 'selectedActivity' implicitly has an 'any' type.
src/Store.ts(163,9): error TS7018: Object literal's property 'user' implicitly has an 'any' type.
src/Store.ts(164,9): error TS7018: Object literal's property 'bot' implicitly has an 'any' type.
src/Store.ts(223,9): error TS7018: Object literal's property 'activities' implicitly has an 'any[]' type.
src/Store.ts(226,9): error TS7018: Object literal's property 'selectedActivity' implicitly has an 'any' type.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! botframework-webchat@0.10.6 build: `tsc && npm run build-css && webpack --progress`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the botframework-webchat@0.10.6 build script 'tsc && npm run build-css && webpack --progress'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the botframework-webchat package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     tsc && npm run build-css && webpack --progress
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs botframework-webchat
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls botframework-webchat
npm ERR! There is likely additional logging output above.
```

**Comment by billba** [comment]
Fixed by #426, sorry about that.

**Comment by bpug** [comment]
Thank you for the fast answer!
